tests/Resources/ti.ui.window.test.js

```diff
@@ -96,6 +96,86 @@ describe('Titanium.UI.Window', function () {
 		});
 	});
 
+	it.ios('#leftNavButtons and #rightNavButtons', finish => {
```

Comment:
```suggestion
	it.ios('.leftNavButtons and .rightNavButtons', finish => {
```

I use the `#` prefix for methods, `.` for properties.